psydac/api/fem.py

```diff
@@ -187,24 +187,38 @@ def __init__(self, expr, kernel_expr, *args, **kwargs):
         # ...
         if len(domain)>1:
             i,j = self.get_space_indices_from_target(domain, target )
-            test_space  = self.spaces[0].spaces[i]
-            trial_space = self.spaces[1].spaces[j]
+            trial_space  = self.spaces[0].spaces[j]
+            test_space = self.spaces[1].spaces[i]
         else:
             trial_space  = self.spaces[0]
             test_space   = self.spaces[1]
 
+        # ...
+        test_ext  = None
+        trial_ext = None
         if isinstance(target, Boundary):
-            axis      = target.axis
-            test_ext  = target.ext
-            trial_ext = target.ext
+            axis        = target.axis
+            ext         = target.ext
+            test_ext    = ext
+            trial_ext   = ext
```

Comment:
What is the point of these changes? You wanted to call the property `target.ext` only once?

Reply:
ah there is no point, maybe I was debugging some stuff and I forgot to change it back

Reply:
done in [4c54406](https://github.com/pyccel/psydac/pull/160/commits/4c5440653d00be8d85aa7c35a2b9c66927667b5a)